lib/application-processing/resolvers.ts

```diff
@@ -767,7 +782,7 @@ export const completeApplication: Resolver<
             firstName,
             lastName,
             updatedApplicant.dateOfBirth,
-            id
+            id.toString()
```

Comment:
The replacement wallet cards should also use the applicant id for “User #” instead of the application id
```suggestion
            applicantId.toString()
```

Reply:
Great catch, updated!